css/component/m3/m3-checkbox.css

```
@@ -0,0 +1,152 @@
+/* 체크박스 변수 */
+.m3-checkbox {
+    --checkbox-main-color: var(--color-main-2);
+    --checkbox-icon-color: var(--color-white);
+    --checkbox-border-color: var(--color-base-8);
+    --checkbox-outline-color: var(--color-main-2);
+    --chckbox-text-color: var(--color-base-8);
```

```suggestion
    --checkbox-text-color: var(--color-base-8);
```
오타로 보입니다! 본문 내 chckbox 모두 수정 바랍니다.